### ISSUE TITLE
Feature Wish: export/show list of all items in search

### ISSUE THREAD

**Issue by Bellfalasch** [GlennRicaud/data-toolbox-app#64]
We rely a lot on search function of this app to find parts that are used in a specific way, before making breaking changes. Example: "Part A allows empty field 1, we will now begin enforcing field 1, wish pages does editors need to go to to fix this?", or "Part B had option for illustration, this will now be replaced with option for photo instead".

The search-function gives us all pages, but communicating a list of simple URLs, like pasted bellow, is very cumbersome. Usually editors will work in teams of 3-5 people on a task when we decide to do breaking data changes like this.

A button that opens a modal with a list of all URLs in the result would get us far, or other similar solution. This type of ist is easy to communicate to a bigger editorial team and get them started on the job manually fixing all those pages and parts.

```
/fragments/fragment-notice-short-text_hemforsakring_uspar
/fragments/fragment-oppettider-kundservice
/fragments/kontakt_hgf
/fragments/pluspaket_bil
/fragments/fragment-foretag-sjalvrisk-ersattning
/under-construction/travel-claims-core/resa-larmcentral
/fragments/fragment-sjalvrisk-ersattning
/fragments/fragment-notice-short-text
/privat/bilforsakring/trafikforsakring
/privat/hemforsakring/smabatsforsakring
/privat/bilforsakring/pluspaketet
/partners/okq8-webshop-bil
/partners/villaagarna_webshop
/foretag/tjanstereseforsakring
/privat/bilforsakring/halvforsakring
/privat/bilforsakring/helforsakring
/privat/motorcykelforsakring/halvforsakring
/privat/skador/person-ishockey
/privat/skador/person-hockey-en
/privat/motorcykelforsakring/trafikforsakring
/privat/motorcykelforsakring/helforsakring
/privat/bilforsakring/hogre-sjalvrisk
/foretag/fordonsforsakring/budbil
/privat/skador/bil2
/privat/skador/bil-logga-in-och-anmal-skada
/foretag/sjukvardsforsakring
/under-construction/villaforsakring-copy-material-action
```

**Comment by GlennRicaud** [comment]
Could it be solved by a new type of Node Search report format?

For example "a CSV file" and being able to specify the fields that should be included in this report. For example in your case: `_path`

Example of CSV file that would be generated:
```
path
/content/superhero
/content/superhero/search
```

**Comment by GlennRicaud** [comment]
Will be available in DT 5.0.21

<img width="1148" alt="Screen Shot 2021-12-14 at 21 59 32" src="https://user-images.githubusercontent.com/12030113/146080193-5b7f162d-2384-4573-897d-f98fddc22c52.png">
<img width="891" alt="Screen Shot 2021-12-14 at 21 59 57" src="https://user-images.githubusercontent.com/12030113/146080208-b5ce31ce-0207-45c1-92a4-3f034c30c884.png">

**Comment by Bellfalasch** [comment]
OMG ... I'm just speechless
